tests/drivers/timer/nrf_grtc_timer/src/main.c

```diff
@@ -48,4 +60,75 @@ ZTEST(nrf_grtc_timer, test_get_ticks)
 	}
 }
 
+ZTEST(nrf_grtc_timer, test_timer_count_in_compare_mode)
+{
+	int err;
+	uint64_t test_ticks = 0;
+	uint64_t compare_value = 0;
+	char user_data[] = "test_timer_count_in_compare_mode\n";
```

Comment:
nit: this `\n` don't seem to be needed as this data is used this way: https://github.com/zephyrproject-rtos/zephyr/blob/fc8d0a85d4d129cb45e2f14b2924ca6c70d6ae27/tests/drivers/timer/nrf_grtc_timer/src/main.c#L23
similarly in `test_timer_abort_in_compare_mode`